templates/engage/casestudy-itstratagen.html

```diff
@@ -0,0 +1,30 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}ITstrategen keeps legacy applications secure with Extended Security Maintenance{% endblock %}
```

Comment:
for the new engage pages we are supposed to use {% extends_with_args .... implemented by David https://github.com/canonical-web-and-design/ubuntu.com/pull/5255/files